ports/rp2/Makefile

```diff
@@ -29,17 +29,10 @@ all:
 clean:
 	$(RM) -rf $(BUILD)
 
-GIT_SUBMODULES += lib/mbedtls lib/tinyusb
```

Comment:
I agree it's for the best to move this all into cmake and always run it through that - having two separate code paths was a rather foolish move on my part!